crates/core/component/auction/src/component/dutch_auction.rs

```diff
@@ -184,6 +367,58 @@ pub(crate) trait DutchAuctionManager: StateWrite {
 
 impl<T: StateWrite + ?Sized> DutchAuctionManager for T {}
 
+#[async_trait]
+pub(crate) trait HandleDutchTriggers: StateWrite {
+    /// Process the trigger height for a [`DutchAuction`],
+    async fn process_trigger(&mut self, trigger_height: u64) -> Result<()> {
+        use futures::StreamExt;
+        // TODO(erwan): somewhat self-defeating, but this is a first pass.
+        let auction_ids: Vec<AuctionId> = self
+            .stream_dutch_ids_by_trigger(trigger_height)
+            .await
+            .collect()
+            .await;
+
+        for auction_id in auction_ids.into_iter() {
+            let _ = self
+                .execute_dutch_auction(auction_id, trigger_height)
+                .await?;
```

Comment:
this looks like we're discarding a result but on closer inspection it seems like `_ = ()`? if so could we drop the let _ = ... so that we don't mislead a reader into thinking that that's the case, or accidentally drop an error we care about in the future?